decapodes/configuration/decapodes.it

```diff
@@ -0,0 +1,42 @@
+struct Dimensionality
+	surface::Int64
+	embedding::Int64
+end
+
+@sum Region begin
+	Equation(equation::String)
+end
+
+struct SubmeshRelation
+  parent::String
+  child::String
+end
+
+struct Mesh
+  id::String
+  description::String
+  file::String
+  checksum::String
+  submesh_relation::Vector{SubmeshRelation}
```

Comment:
I think that you probably want the SubmeshRelation to live outside a specific mesh, because it is a binary relation on the set of Regions. You should think of the submesh relation as a graph where each containment is an edge.